src/main/java/sirius/biz/storage/s3/ObjectStore.java

```diff
@@ -714,6 +717,8 @@ public void upload(BucketName bucket, String objectId, InputStream inputStream)
         ensureBucketExists(bucket);
         InitiateMultipartUploadResult multipartUpload =
                 getClient().initiateMultipartUpload(new InitiateMultipartUploadRequest(bucket.getName(), objectId));
+
+        boolean emptyInputStream = false;
```

Comment:
I would rename the variable to: isInputStreamEmpty

Reply:
Or better: describe from which the input stream originates or what the bool is actualy there to check for

Reply:
Why should the name of the boolean start with `is`? I find this a good rule of thumb for a boolean getter method name, but in the actual variable/field name this is unnecessary noise